week-1-practise/level-3/1-piping.js

```diff
@@ -11,6 +11,7 @@
 */
 
 var startingValue = 5;
+var result = (startingValue * 2 + 10).toFixed(2);
```

Comment:
Please try and use the provided functions to `add`, `multiply`, and `format`

Reply:
I really don't understand this exercise. I read some stuff about piping functions but I still don't get it :(